pkgs/build-support/rust/fetch-cargo-vendor.nix

```diff
@@ -0,0 +1,93 @@
+{
+  lib,
+  stdenvNoCC,
+  runCommand,
+  writers,
+  python3Packages,
+  cargo,
+  nix-prefetch-git,
+  cacert,
+}:
+
+let
+  replaceWorkspaceValues = writers.writePython3Bin "replace-workspace-values" {
+    libraries = with python3Packages; [
+      tomli
+      tomli-w
```

Comment:
python also can parse toml nativly now. It's only needed for writing, which doesn't seem to be used here?

Reply:
`tomli-w` is only used by `replace-workspace-values.py`, which I didn't write.
I migrated over to `tomllib` for the vendor script.